### PR TITLE
Add mnist test for post training quantization, test=develop

### PR DESCRIPTION
**Pull request by juncaipeng**
### PR types

Others
### PR changes

Others
### Describe

Add mnist test for post training quantization, test=develop

### CODE REVIEW

File: python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+import contextlib
+import numpy as np
+import paddle
+import paddle.fluid as fluid
+from paddle.dataset.common import download
+from paddle.fluid.contrib.slim.quantization import PostTrainingQuantization
+
+random.seed(0)
+np.random.seed(0)
+os.environ["CUDA_VISIBLE_DEVICES"] = "0"
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py]: This is not needed and useless.

File: python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    def test_post_training_kl(self):
+        model_name = "mnist_model"
+        data_url = "http://paddle-inference-dist.bj.bcebos.com/int8/mnist_model.tar.gz"
+        data_md5 = "be71d3997ec35ac2a65ae8a145e2887c"
+        algo = "KL"
+        quantizable_op_type = ["conv2d", "depthwise_conv2d", "mul"]
+        is_full_quantize = False
+        is_use_cache_file = False
+        is_optimize_model = True
+        diff_threshold = 0.02
```
Comment (wzzju) [inline python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py]: Why not set 0.01?

**Review by wzzju — DISMISSED** [review]

File: python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py (hunk 1 above)
Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py]: Done

File: python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py (hunk 2 above)
Comment (juncaipeng) [reply python/paddle/fluid/contrib/slim/tests/test_post_training_quantization_mnist.py]: Done

**Review by wzzju — COMMENTED** [review]
LGTM.

**Review by wzzju — APPROVED** [review]
LGTM.

**Pull request merged**